src/service/standard-actions-impl.js

```diff
@@ -68,9 +68,38 @@ export class StandardActions {
     /** @const @private {!./viewport/viewport-impl.Viewport} */
     this.viewport_ = Services.viewportForDoc(ampdoc);
 
+    /** @const @private {?Array<string>} */
+    this.ampActionWhitelist_ = null;
+
     this.installActions_(this.actions_);
   }
 
+  /**
+   * Searches for a meta tag containing whitelist of actions on
+   * the special AMP target, e.g.,
+   * <meta name="amp-action-whitelist" content="AMP.setState,AMP.pushState">
+   * @return {?Array<string>} the whitelist of actions on the
+   * special AMP target.
```

Comment:
Doesn't the text here fit on the previous line? If not, this line should be indented 4 space, starting from the column where the `@` on the previous line is.